util/journaldb/src/overlayrecentdb.rs

```diff
@@ -70,12 +70,38 @@ use crate::{
 /// the removed key is not present in the history overlay.
 /// 7. Delete ancient record from memory and disk.
 
+const CACHE_MEM_USED_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(60);
```

Comment:
Seems rather arbitrary, maybe expose this as a config flag?

Reply:
Based on some preliminary estimations, scanning 128GB it will take 30s, so I just doubled it. Nonetheless, yes, we can expose as a config flag.